src/main/app/drafts/models/draftClaim.ts

```diff
@@ -21,6 +22,7 @@ export default class DraftClaim extends DraftDocument implements Serializable<Dr
   readResolveDispute: boolean = false
   readCompletingClaim: boolean = false
   qualifiedStatementOfTruth?: QualifiedStatementOfTruth
+  eligibility: Eligibility = new Eligibility()
```

Comment:
will try remove it, I was having problems in the route before

Reply:
Its needed because in DraftClaim there's a check for no input and if there's no input it skips the deserialising.

The no input happens when there's nothing in the draft yet.

Could probably change it so that there's always a draft but that's unrelated to this.